examples/kyb-app/DynamicElements/hooks/useContext/useContext.ts

```diff
@@ -0,0 +1,37 @@
+import { UIElement } from '@app/domains/collection-flow';
+import { AnyObject } from '@ballerine/ui';
+import { useCallback, useEffect, useRef, useState } from 'react';
+
+export const useContext = <TContext extends AnyObject>(
```

Comment:
Let's think of better naming here. Context has meaning as part of React and `useContext` is too general considering.